validation/checks/ark_check.go

```diff
@@ -1,5 +1,3 @@
-// This file checks that ARK values are valid according to the spec.
```

Comment:
I'm guessing you put the package level docs here because it was the first file, alphabetically, in the package, but since "which file sorts first" could change over time, I think I'd recommend we put the package level docs on the setup_test.go file since that will be common across all packages (so we know where to look for the docs), but... it also looks like the checks package doesn't have one of those? I think every other package does though. I must have missed the checks package.